bin/util_docker/builder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.9.2 AS builder
```

Comment:
Would be nice to use this new image in `bin/server_docker/ubuntu/Dockerfile` etc.

Reply:
changed; tested